data/beaconrestapi/src/test/java/tech/pegasys/artemis/beaconrestapi/BeaconRestApiTest.java

```diff
@@ -31,57 +33,51 @@
 import tech.pegasys.artemis.storage.ChainStorageClient;
 
 class BeaconRestApiTest {
-  private final ChainStorageClient storageClient =
+  private static final ChainStorageClient storageClient =
       ChainStorageClient.memoryOnlyClient(new EventBus());
-  private final JavalinServer server = mock(JavalinServer.class);
-  private final Javalin app = mock(Javalin.class);
+  private static final JavalinServer server = mock(JavalinServer.class);
+  private static final Javalin app = mock(Javalin.class);
   private static final Integer THE_PORT = 12345;
 
-  @Test
-  public void RestApiShouldHaveServerPortSet() {
+  @BeforeAll
+  public static void setup() {
+
     when(app.server()).thenReturn(server);
     new BeaconRestApi(storageClient, null, null, THE_PORT, app);
+  }
```

Comment:
I suspect you want `@BeforeEach` so the each test remains independent.

Reply:
changed to BeforeEach